Script/AtomicEditor/ui/frames/menus/MainFrameMenu.ts

```diff
@@ -40,7 +40,8 @@ class MainFrameMenu extends Atomic.ScriptObject {
         MenuItemSources.createMenuItemSource("menu tools", toolsItems);
         MenuItemSources.createMenuItemSource("menu developer", developerItems);
         MenuItemSources.createMenuItemSource("menu help", helpItems);
-
+        this.goScreenshot = 0;
+        this.subscribeToEvent(Atomic.UpdateEvent((ev) => this.handleScreenshot(ev)));
```

Comment:
@shaddockh I think we can require the event loop methods for the editor in main.ts can't we, then they are accessible in the VM at global scope: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/AtomicEditor/main.ts

The event loop is in CoreData, so available to all Atomic apps, editor and players included :) 

https://github.com/AtomicGameEngine/AtomicGameEngine/tree/master/Resources/CoreData/AtomicModules

@JimMarlowe setTimeout is part of the standard JS API provided by requiring AtomicEventLoop